xml/System.Windows.Forms/ToolStripDropDownButton.xml

```diff
@@ -232,9 +232,19 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Creates a new accessibility object for this <see cref="T:System.Windows.Forms.ToolStripDropDownButton" /> instance.</summary>
```

Comment:
A couple of issues with this:
- It's much too long for a summary. Much of the text belongs in the Remarks section.
- What "better properties" are is unclear.

Reply:
I don't understand the second bullet point. Improvements are described in the remarks section.

Reply:
@Tanya-Solyanik, you're right. I seem to have been looking at the remarks and thought they were the member description.

Reply:
So should I just discard this feedback then?

Reply:
@mairaw, yes.

Reply:
Now that I'm reviewing again, I think you meant the summary for the MonthCalendar @rpetrusha

Reply:
Yes, you're right, @mairaw. It is the summary for MonthCalendar; I apologize that I somehow attached my comment to a different summary.  @Tanya-Solyanik, can we phrase this to avoid the use of the word "better"?

Reply:
Sure, remarks for the other control were perfect, can we poach them?

Reply:
My suggestion for MonthCalendar.CreateAccessibilityObject remarks:

The AccessibleObject (https://msdnstage.redmond.corp.microsoft.com/en-US/library/system.windows.forms.accessibleobject(v=vs.110).aspx)  instance returned by this method provides correct values for Help, Name, and Role properties. The new value for Role property is AccessibleRole.Table, new value for Name property is the current selection, and the new default value for the  Help property is type name. However the new object is only available in applications that are recompiled to target .NET Framework 4.7.1 or opt-in into this feature using a compatibility switch. For more information, see the Windows Forms section in the Retargeting Changes for Migration to the .NET Framework 4.7.1 topic.